servers/src/test/java/tachyon/master/file/FileSystemMasterTest.java

```diff
@@ -95,6 +95,7 @@ public static void beforeClass() {
   @AfterClass
   public static void afterClass() {
     TtlBucketPrivateAccess.setTtlIntervalMs(sOldTtlIntervalMs);
+    MasterContext.reset();
```

Comment:
this should go in an `@After` method since the conf is modified in the `@Before`